repolint.json

```diff
@@ -4,316 +4,316 @@
+        "type": "directory-existence",
+        "options": {
+          "globsAny": [
+            "**/test*",
+            "**/specs",
+            "**/*test*"
+          ],
+          "nocase": true
         }
+      }
     },
     "integrates-with-ci": {
-        "level": "warning",
-        "rule": {
-            "type": "file-existence",
-            "options": {
-                "globsAny": [
-                    ".gitlab-ci.yml",
-                    ".travis.yml",
-                    "appveyor.yml",
-                    ".appveyor.yml",
-                    "circle.yml",
-                    ".circleci/config.yml",
-                    "Jenkinsfile",
-                    ".drone.yml",
-                    ".github/workflows/*",
-                    "azure-pipelines.yml"
-                ]
-            }
+      "level": "warning",
+      "rule": {
+        "type": "file-existence",
+        "options": {
+          "globsAny": [
+            ".gitlab-ci.yml",
+            ".travis.yml",
+            "appveyor.yml",
+            ".appveyor.yml",
+            "circle.yml",
+            ".circleci/config.yml",
+            "Jenkinsfile",
+            ".drone.yml",
+            ".github/workflows/*",
+            "azure-pipelines.yml"
+          ]
         }
+      }
     },
     "source-license-headers-exist": {
-        "level": "error",
-        "rule": {
-            "type": "file-starts-with",
-            "options": {
-                "globsAll": [
-                    "**/*.py",
-                    "**/*.js",
-                    "**/*.c",
-                    "**/*.cc",
-                    "**/*.cpp",
-                    "**/*.h",
-                    "**/*.ts"
-                ],
-                "skip-paths-matching":{
-                    "patterns": [
-                        "babel.config.js",
-                        "build/*",
-                        "jest.config.js",
-                        "node_modules/*",
-                        "types/*"
-                    ]
-                 },
-                "lineCount": 40,
-                "patterns": [
-                    "Copyright.*Qualcomm Innovation Center, Inc",
-                    "SPDX-License-Identifier"
-                ],
-                "flags": "i"
-            }
+      "level": "error",
+      "rule": {
+        "type": "file-starts-with",
+        "options": {
+          "globsAll": [
+            "**/*.py",
+            "**/*.js",
+            "**/*.c",
+            "**/*.cc",
+            "**/*.cpp",
+            "**/*.h",
+            "**/*.ts"
+          ],
+          "skip-paths-matching": {
+            "patterns": [
+              "babel.config.js",
+              "build/*",
+              "jest.config.js",
+              "node_modules/*",
+              "types/*"
+            ]
+          },
+          "lineCount": 40,
+          "patterns": [
+            "Copyright.*Qualcomm Innovation Center, Inc|© 2021 Qualcomm Innovation Center, Inc",
```

Comment:
Why include the year? Won't that cause issues in the future?

Suggest instead:

`© .*Qualcomm`

Reply:
Thanks Craig. I had done that in testing but forgot to commit it.